_dprhtml/js/format.js

```diff
@@ -147,6 +149,9 @@ function formatuniout(sectionId, data,which) { // which = 1 prepare without link
 
   var wb;
   var b = 0;
+  if (which == 3) { // from search results page

```

Comment:
Please add the comment for which next to the 2 other values in the function definition line.

Reply:
OK done